modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx

```diff
@@ -48,6 +51,40 @@ export function EmailNotificationSettings({
 				translations={(values.recipients[0] as EmailRecipients).to}
 			/>
 
+			<ClayForm.Group className="ml-1 row">
+				<div className="mr-2">
+					<ClayCheckbox
+						checked={
+							(values.recipients[0] as EmailRecipients)
+								.singleRecipient
+						}
+						disabled={false}
```

Comment:
@seltong The ClayCheckbox has the property `disabled` as `false` by default, so you don't need to pass it.

Reply:
Yeah, it makes sense. I saw similar cases in some files like [this](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/BasicInfoScreen/BasicInfoScreen.tsx#L64-L73) and though that it was a pattern or something like this.
I will fix this in these files too, ok?